### DISCUSSION TITLE
Print output of unknown origin or purpose

### DISCUSSION THREAD

**Discussion by michaelosthege** [pydata/xarray#5628] [Q&A] [answered]
Hi all,

while building a new PyMC3 model, I got some print output of unknown origin.
I couldn't find any "?=" in the code of PyMC3 or ArviZ, so in the PyMC team we suspect that it might come from xarray?

```
0, dim: sampling_cycle, 5 =? 5
1, dim: calibration_well, 18 =? 18
0, dim: sampling_cycle, 5 =? 5
1, dim: calibration_well, 18 =? 18
0, dim: sampling_cycle, 5 =? 5
```

Does that pattern look familiar to anybody?
thanks!

**Comment by max-sixty** [comment, marked as answer]
I don't think I've ever seen this. I think we're also fairly careful for arbitrary print statements, though one could have crept through. Do you have a repro?

**Reply by michaelosthege** [reply]
Nevermind. It looks like it was a rogue print statement in ArviZ that made its way into a release, but was since removed, which is why my search didn't find it.

Thanks for confirming that it wasn't from xarray though!
